flutter/lib/resources/cache_manager.dart

```diff
@@ -56,6 +64,15 @@ class CacheManager {
     }
   }
 
+  Future<void> deleteArchives(List<String> resources) async {
+    for (final resource in resources) {
+      final archivePath = getArchive(resource);
+      if (archivePath == null) continue;
+      final archiveFile = File(archivePath);
+      if (archiveFile.existsSync()) await archiveFile.delete();
```

Comment:
Is there a reason to use `existsSync` here instead of `exists`?

Reply:
We can use `await file.exists()` here but I don't see any difference.

Reply:
`existsSync` blocks the thread.
Since we are running almost everything in the UI thread blocking it makes the UI unresponsive.
Well, most of the time it will probably be fine, but if the storage does not respond for any reason delays could get noticeable.

Reply:
It's not a major issue by any means but I don't see any reason to take additional risks.

Reply:
Ok.